src/BuildingRegistry.Producer/Extensions/MessageExtensions.cs

```diff
@@ -518,6 +518,14 @@ public static Contracts.BuildingWasRemovedV2 ToContract(this BuildingWasRemovedV
                 message.BuildingPersistentLocalId,
                 message.Provenance.ToContract());
 
+        public static Contracts.BuildingUnitAddressWasReplacedBecauseAddressWasReaddressed ToContract(this BuildingUnitAddressWasReplacedBecauseAddressWasReaddressed message)
+            => new Contracts.BuildingUnitAddressWasReplacedBecauseAddressWasReaddressed(
+                message.BuildingPersistentLocalId,
+                message.BuildingUnitPersistentLocalId,
+                message.PreviousAddressPersistentLocalId,
+                message.NewAddressPersistentLocalId,
+                message.Provenance.ToContract());
+
```

Comment:
Constructor has the newAddressPersistentLocalId before the previousAddressPersistentLocalId
https://github.com/Informatievlaanderen/grar-common/blob/37ffba84b06b55a294b92a5a884f9ff2e3cc0a6b/src/Be.Vlaanderen.Basisregisters.GrAr.Contracts/BuildingRegistry/BuildingUnitAddressWasReplacedBecauseAddressWasReaddressed.cs